test/integration/examples/apiserver_test.go

```diff
@@ -131,7 +138,7 @@ func TestAggregatedAPIServer(t *testing.T) {
 
 	// wait for the unavailable API service to be processed with updated status
 	err = wait.Poll(100*time.Millisecond, 5*time.Second, func() (done bool, err error) {
-		_, err = kubeClient.Discovery().ServerResources()
+		_, _, err = kubeClient.Discovery().ServerGroupsAndResources()
```

Comment:
why is this changed?

Reply:
This is a static check error. `ServerResources` has been deprecated so we'd use `ServerGroupsAndResources` instead.